doc/changelog.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 Changes in Version 4.2
 ----------------------
 
+.. warning:: PyMongo 4.2 drops support for Python 3.6, Python 3.7+ is now required.
```

Comment:
I think this would read clearer with a semicolon after 3.6.  I had to read it twice.

Reply:
Done.